be/src/exec/pipeline/aggregate/aggregate_blocking_sink_operator.cpp

```diff
@@ -43,6 +43,12 @@ void AggregateBlockingSinkOperator::close(RuntimeState* state) {
 }
 
 Status AggregateBlockingSinkOperator::set_finishing(RuntimeState* state) {
+    // skip processing if cancelled
+    if (state->is_cancelled()) {
+        _is_finished = true;
+        return Status::Cancelled("runtime state is cancelled");
+    }
```

Comment:
_is_finished = true; should be set?

Reply:
it doesn't matter. cancel has indenpendent processing.